crates/optimism/node/src/node.rs

```diff
@@ -519,6 +541,24 @@ where
         let Self { pool_config_overrides, .. } = self;
         let data_dir = ctx.config().datadir();
         let blob_store = DiskFileBlobStore::open(data_dir.blobstore(), Default::default())?;
+        // Interop check
+        let supervisor_client;
+        if ctx.chain_spec().is_interop_active_at_timestamp(ctx.head().timestamp) {
```

Comment:
I believe this setup is problematic, because this doesn't account for when interop will be activated and only check the current local head block, which will always be false if a fresh node is started.

we need to track activation dynamically:

https://github.com/paradigmxyz/reth/blob/9cb7177dfc692111888068f52b9b89db0fce2177/crates/transaction-pool/src/validate/eth.rs#L114-L117

similar to how fork activations are tracked here:

https://github.com/paradigmxyz/reth/blob/9cb7177dfc692111888068f52b9b89db0fce2177/crates/transaction-pool/src/validate/eth.rs#L519-L527